services/QuillLMS/db/migrate/20250306185139_create_classlink_accounts.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateClasslinkAccounts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :classlink_accounts do |t|
+      t.string :external_id, null: false
+      t.references :user, foreign_key: true, null: false
+      t.references :classlink_instance, foreign_key: true, null: false
```

Comment:
Not an objection, but we've tended to be cautious about declaring `foreign_key` values due to the additional overhead it imposes on database operations.  Are we getting enough value out of the extra validation here to be worth the computational overhead?